### PR TITLE
cap1xxx - add support to specify alternate i2c address (#111)

### PR DESCRIPTION
**Pull request by Gadgetoid**
The cap1xxx range support multiple i2c addresses which are used across our various boards- GFX HAT, Display-o-tron HAT, Drum HAT, Piano HAT, etc- this patch adds support for specifying an i2c address

See https://github.com/androidthings/contrib-drivers/issues/111 for details.

### CODE REVIEW

File: cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxx.java [hunk 1]
```diff
@@ -213,22 +213,38 @@ public Cap1xxx(Context context, String i2cName, String alertName, Configuration
...
+     */
+    public Cap1xxx(String i2cName, String alertName, Configuration chip, Handler handler) throws IOException {
+        this(i2cName, I2C_ADDRESS, alertName, chip, handler);
+    }
+
     /**
      * Create a new Cap1xxx controller.
      *
      * @param i2cName I2C port name where the controller is attached. Cannot be null.
+     * @param i2cAddress 7-bit I2C address for the attached controller. Cannot be null.
```
Comment (devunwired) [inline cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxx.java]: int values cannot be null, I'd remove that phrase from the end.

File: cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxxInputDriver.java [hunk 2]
```diff
@@ -100,6 +100,25 @@ public Cap1xxxInputDriver(String i2cName, String alertName, Configuration chip,
         init(peripheral, keyCodes);
     }
 
+    /**
+     * Create a new Cap1xxxInputDriver to forward capacitive touch events to the Android input
+     * framework.
+     *
+     * @param i2cName I2C port name where the controller is attached. Cannot be null.
+     * @param i2cAddress 7-bit I2C address for the attached controller. Cannot be null.
```
Comment (devunwired) [inline cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxxInputDriver.java]: int values cannot be null, I'd remove that phrase from the end.

File: cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxx.java [hunk 3]
```diff
@@ -213,22 +213,38 @@ public Cap1xxx(Context context, String i2cName, String alertName, Configuration
...
+     *
+     * @param i2cName I2C port name where the controller is attached. Cannot be null.
+     * @param alertName optional GPIO pin name connected to the controller's
+     *                  alert interrupt signal. Can be null.
+     * @param chip identifier for the connected controller device chip.
+     * @param handler optional {@link Handler} for software polling and callback events.
+     * @throws IOException
+     */
+    public Cap1xxx(String i2cName, String alertName, Configuration chip, Handler handler) throws IOException {
+        this(i2cName, I2C_ADDRESS, alertName, chip, handler);
```
Comment (devunwired) [inline cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxx.java]: This is a good change, but if we expose the I2C address as a parameter let's rename the constant to `DEFAULT_I2C_ADDRESS` to match our other drivers ([example](https://github.com/androidthings/contrib-drivers/blob/master/bmx280/src/main/java/com/google/android/things/contrib/driver/bmx280/Bmx280.java#L48))

File: cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxx.java [hunk 4]
```diff
@@ -192,7 +192,7 @@ public Cap1xxx(Context context, String i2cName, String alertName, Configuration
     }
 
     /**
-     * Create a new Cap1xxx controller.
+     * Create a new Cap12xx controller with the default I2C address.
```
Comment (devunwired) [inline cap1xxx/src/main/java/com/google/android/things/contrib/driver/cap1xxx/Cap1xxx.java]: Rename to Cap1xxx

**Comment by Gadgetoid** [comment]
I believe all requested changes are made @devunwired

**Pull request merged**